### PR TITLE
change: added forum links

### PR DESCRIPTION
**Pull request by meathill**

### CODE REVIEW

File: v2/en/community.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
     @created       2012-08-04 23:04 GMT
 --->
 
+# Official Forum
+
+We have established an official forum and imported the contents from mailing list (as of 2020-03-28). Welcome to visit:
```
Comment (agentzh) [inline v2/en/community.md:9]: “established" => "set up"

"contents" => "archive"

"mailing list" => "mailing lists"

"2020-03-28" => "March 28, 2020"

"Welcome to visit" => "Welcome to join us there"

File: v2/en/contact-us.md [hunk 2]
```diff
@@ -8,9 +8,22 @@
...
 --->
 
+Official forum
+--------
+
+We have established an official forum and imported the contents from mailing list (as of 2020-03-28). Welcome to visit:
+
+[https://forum.openresty.us/](https://forum.openresty.us/)
+
+Mailing list
```
Comment (agentzh) [inline v2/en/contact-us.md:18]: "list" => "lists"

File: v2/en/contact-us.md [hunk 3]
```diff
@@ -8,9 +8,22 @@
...
+--------
+
 You can join the English or Chinese mailing lists shown on the page [OpenResty Community](community.html).
 This is usually the preferred communication channel.
 
+Other Contact
```
Comment (agentzh) [inline v2/en/contact-us.md:24]: "Other Contact" => "Contact us directly"

File: v2/en/contact-us.md [hunk 4]
```diff
@@ -8,9 +8,22 @@
...
+--------
+
 If you need private communication, please contact us by emails via `info@openresty.org`.
 
 Reporting bugs are encouraged to file tickets for the ngx_openresty project
```
Comment (agentzh) [inline v2/en/contact-us.md:29]: Let's change "ngx_openresty" to OpenResty as well :)

**Comment by meathill** [comment]
@agentzh changed. Please check again.

**Comment by agentzh** [comment]
Merged after removing the forum from the English pages since they look too Chinese now.

**Pull request closed without merging**